packages/lab/src/calendar/internal/CalendarDay.css

```diff
@@ -1,163 +1,99 @@
-}
-
-.saltCalendarDay-hoveredSpan,
-.saltCalendarDay-hoveredSpan:hover {
-  background: var(--calendar-day-hoveredSpan-background);
-  color: var(--calendar-day-hoveredSpan-text-color);
+  color: var(--salt-content-secondary-foreground);
 }
 
+.saltCalendarDay:hover,
 .saltCalendarDay-hoveredOffset,
 .saltCalendarDay-hoveredOffset:hover {
-  background: var(--calendar-day-hoveredOffset-background);
-  color: var(--calendar-day-hoveredOffset-text-color);
+  background: var(--salt-selectable-background-hover);
 }
-
-.saltCalendarDay-selected,
-.saltCalendarDay-selected:hover {
-  background: var(--calendar-day-selected-background);
-  color: var(--calendar-day-selected-text-color);
+.saltCalendarDay-selectedStart {
+  z-index: var(--salt-zIndex-default);
 }
 
+.saltCalendarDay-selected,
+.saltCalendarDay-selected:hover,
 .saltCalendarDay-selectedStart,
-.saltCalendarDay-selectedStart:hover {
-  background: var(--calendar-day-selectedStart-background);
-  color: var(--calendar-day-selectedStart-text-color);
-}
-
+.saltCalendarDay-selectedStart:hover,
 .saltCalendarDay-selectedEnd,
 .saltCalendarDay-selectedEnd:hover {
-  background: var(--calendar-day-selectedEnd-background);
-  color: var(--calendar-day-selectedEnd-text-color);
-}
-
-.saltCalendarDay-selected:focus-visible {
-  outline-color: var(--calendar-day-selected-focused-outlineColor);
-}
-
-.saltCalendarDay-selectedStart:focus-visible {
-  outline-color: var(--calendar-day-selectedStart-focused-outlineColor);
-}
-
-.saltCalendarDay-selectedEnd:focus-visible {
-  outline-color: var(--calendar-day-selectedEnd-focused-outlineColor);
-}
-
-.saltCalendarDay-unselectableLow,
-.saltCalendarDay-unselectableLow:hover {
-  color: var(--calendar-day-unselectable-text-color);
-  background: var(--calendar-day-unselectable-background);
-  cursor: var(--calendar-day-unselectable-cursor);
-  text-decoration: line-through;
+  background: var(--salt-selectable-background-selected);
+  box-shadow: 0 0 0 var(--salt-size-border) var(--salt-selectable-borderColor-selected);
 }
 
-.salt-density-high {
-  --calendar-day-blocked-icon-size: 14px;
+.saltCalendarDay-hoveredSpan,
+.saltCalendarDay-hoveredSpan:hover,
+.saltCalendarDay-selectedSpan,
+.saltCalendarDay-selectedSpan:hover {
+  background: var(--salt-selectable-background-hover);
+  box-shadow: calc(var(--salt-size-border) * -2) 0 0 0 var(--salt-selectable-background-hover),
+    calc(var(--salt-size-border) * -1) 0 0 var(--salt-size-border) var(--salt-selectable-borderColor-selected), 0 calc(var(--salt-size-border) * -1) 0 var(--salt-selectable-borderColor-selected);
+}
+
+.saltCalendarDay-disabled,
+.saltCalendarDay-disabled:hover,
+.saltCalendarDay-unselectable,
+.saltCalendarDay-unselectable:hover {
+  background: var(--salt-selectable-background-disabled);
+  cursor: var(--salt-selectable-cursor-disabled);
+  position: relative;
 }
-
-.salt-density-medium {
-  --calendar-day-blocked-icon-size: 18px;
+.saltCalendarDay-highlighted {
```

Comment:
It feels weird to only apply this in highlighted. What's the downside to always applying it